bin/Loader_Paths_Finder.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+namespace WP_Syntex\Polylang\Script;
+
+use RuntimeException;
+use Composer\Script\Event;
+
+/**
+ * Class to hard-code `glob()`s.
+ *
+ * @since 3.8
+ */
+class Loader_Paths_Finder {
+	/**
+	 * Hard-codes `glob()`s.
+	 *
+	 * @since 3.8
+	 *
+	 * @throws RuntimeException If arguments are missing or the operation fails.
+	 *
+	 * @param Event $event The Composer event. Expects two arguments.
+	 *     - Absolute path to the folder containing the files to load.
+	 *     - Name of the package.
+	 * @return void
+	 */
+	public static function hard_code( Event $event ): void {
+		$args = $event->getArguments();
+
+		if ( empty( $args[0] ) ) {
+			throw new RuntimeException( 'No path to folder provided' );
+		}
+		if ( empty( $args[1] ) ) {
+			throw new RuntimeException( 'No package name provided' );
+		}
+
+		$base_path   = realpath( $args[0] );
+		$plugin_name = str_replace( '-', ' ', $args[1] );
+		$type        = rtrim( basename( $base_path ), 's' );
+		$file_paths  = glob( "{$base_path}/*/load.php", \GLOB_NOSORT );
+
+		if ( ! is_array( $file_paths ) || empty( $file_paths ) ) {
+			throw new RuntimeException( "Could not retrieve the {$type} files in {$plugin_name}" );
```

Comment:
Allowing to remove the second argument, usign @Hug0-Drelon's suggestion to use the path:
```suggestion
			throw new RuntimeException( "Could not retrieve the files from {$base_path}" );
```